jetbrains-ultimate/src-201+/software/aws/toolkits/jetbrains/datagrip/auth/SecretsManagerAuth.kt

```diff
@@ -61,8 +61,22 @@ class SecretsManagerAuth : DatabaseAuthProvider, CoroutineScope by ApplicationTh
             val project = connection.runConfiguration.project
             try {
                 val connectionSettings = getConfiguration(connection)
-                val credentials = getCredentials(connection.runConfiguration.project, connectionSettings)
-                DatabaseCredentialsAuthProvider.applyCredentials(connection, credentials, true)
+                val dbSecret = getDbSecret(connection.runConfiguration.project, connectionSettings)
+                if (connection.connectionPoint.additionalJdbcProperties[GET_URL_FROM_SECRET]?.toBoolean() == true) {
+                    dbSecret.host ?: throw IllegalArgumentException(message("datagrip.secretsmanager.validation.no_host", connectionSettings.secretId))
```

Comment:
We can't hide it. The error doesn't show the URL at all, it's the friendly message from the database (if it gets there) otherwise it shows "connection refused"